test/libsolidity/util/BytesUtils.cpp

```diff
@@ -206,6 +225,19 @@ string BytesUtils::formatString(bytes const& _bytes, size_t _cutOff)
 	return os.str();
 }
 
+std::string BytesUtils::formatFixedPoint(bytes const& _bytes, size_t _fixedPointM, size_t _fixedPointN)
+{
+	solAssert(_fixedPointM != 0, "");
+	u256 max{boost::multiprecision::pow(u256{2}, static_cast<unsigned>(_fixedPointM))};
+	solAssert(u256{_bytes} >= max, "");
```

Comment:
```suggestion
```
I don't think this is needed.

Reply:
This should prevent using invalid values within the tests.

Reply:
Ok, you're right. This check does not make much sense here, because it's just the formatting function.